Objects/perf_trampoline.c

```diff
@@ -185,6 +201,10 @@ struct trampoline_api_st {
 
 typedef struct trampoline_api_st trampoline_api_t;
 
+#ifdef PY_HAVE_INVALIDATE_ICACHE
+
+#endif
```

Comment:
What is the purpose of this code? Does it optimize invisible code? :-)

Reply:
Is invisible code. Very important for performance